website/cue/reference/remap/functions/parse_url.cue

```diff
@@ -4,6 +4,8 @@ remap: functions: parse_url: {
 	category:    "Parse"
 	description: """
 		Parses the `value` in [URL](\(urls.url)) format.
+
+		The port number will be defaulted if unspecified in the URL and the scheme is one of: `http`, `https`, `ws`, `wss`, and `ftp`, schemes.
```

Comment:
I would suggest to phrase this as:
```
If the port number is not explicit in the input URL string, it will be populated from well-known ports of the following schemes: `http`, `https`, `ws`, `wss`, and `ftp`.
```